charts/datadog/values.yaml

```
@@ -254,6 +254,16 @@ datadog:
   #   env: environment
   #   <KUBERNETES_NAMESPACE_ANNOTATIONS>: <DATADOG_TAG_KEY>
 
+  # datadog.kubernetesResourcesLabelsAsTags -- Provide a mapping of Kubernetes Resources Labels to Datadog Tags
+  kubernetesResourcesLabelsAsTags: {}
+  #   env: environment
+  #   <KUBERNETES_RESOURCE_LABEL>: <DATADOG_TAG_KEY>
+
+  # datadog.kubernetesResourcesAnnotationsAsTags -- Provide a mapping of Kubernetes Resources Annotations to Datadog Tags
+  kubernetesResourcesAnnotationsAsTags: {}
+  #   env: environment
+  #   <KUBERNETES_RESOURCE_ANNOTATIONS>: <DATADOG_TAG_KEY>
```

I think it might worth putting a more accurate example like the one you provided in the PR description:

```suggestion
  # datadog.kubernetesResourcesLabelsAsTags -- Provide a mapping of Kubernetes Resources Labels to Datadog Tags
  kubernetesResourcesLabelsAsTags: {}
  #    deployments.apps:
  #      x-team: team-from-label
  #    pods:
  #      x-ref: reference
  #    namespaces:
  #      kubernetes.io/metadata.name: name-as-tag
  #    nodes:
  #      kubernetes.io/os: os-as-tag
  #      kubernetes.io/arch: arch-as-tag
  #    <RESOURCE_TYPE>:
  #      <KUBERNETES_RESOURCE_LABEL>: <DATADOG_TAG_KEY>

  # datadog.kubernetesResourcesAnnotationsAsTags -- Provide a mapping of Kubernetes Resources Annotations to Datadog Tags
  kubernetesResourcesAnnotationsAsTags: {}
  #    deployments.apps:
  #      x-team: team-from-annotation
  #    pods:
  #      x-ann: annotation-reference
  #    namespaces:
  #      stale-annotation: annotation-as-tag
  #    <RESOURCE_TYPE>:
  #      <KUBERNETES_RESOURCE_ANNOTATION>: <DATADOG_TAG_KEY>
```

---

I totally agree having this more specific example provides more clarity